submissions/UA-girl/task_js_dom/js/script.js

```diff
@@ -0,0 +1,199 @@
+        'The cats have a narrow, long head and webbed feet. Their skin is the color that their fur would be, and all ' +
+        "the usual cat markings (solid, point, van, tabby, tortie, etc.) may be found on the Sphynx cat's skin. " +
+        "Because they have no fur they lose more body heat than coated cats which makes them warm to the touch and prone to finding warm places...",
+        link: 'https://en.wikipedia.org/wiki/Sphynx_cat',
+        image: {
+            url: './img/canadian_sphinx.jpg',
+            alt: 'Canadian Sphynx cat image'
+        }
+    },
+
+    {
+        id: 4,
+        title: 'Turkish Angora',
+        text: "The Turkish Angora (Turkish: Ankara kedisi, 'Ankara cat') is a breed of domestic cat. Turkish Angoras " +
+        "are one of the ancient, natural breeds of cat, having originated in central Turkey, in the Ankara region. " +
+        "The breed has been documented as early as the 17th century and is believed to be the origin of the mutations " +
+        "for both the color white[citation needed] and long hair[citation needed]. " +
+        "The breed is also sometimes referred to as simply the Angora or Ankara cat...",
+        link: 'https://en.wikipedia.org/wiki/Turkish_Angora',
+        image: {
+            url: './img/angorra.jpg',
+            alt: 'Angorra cat image'
+        }
+    },
+];
+const aside = document.querySelector('.aside');
+const card = document.querySelector('.card');
+
+const menu = createMenu();
+createCards();
+
+
+// Fill card
+function createCardPattern({title, text, link, image: {alt, url}}, node) {
+    // Update image
+    const img = node.querySelector(`.${IMG_CLASS_NAME}`);
+    img.setAttribute('alt', alt);
+    img.setAttribute('src', url);
+    // Update data
+    const caption = node.querySelector(`.${CARD_DESCRIPTION_CLASS_NAME}`);
+    caption.querySelector(`.${CARD_TITLE_CLASS_NAME}`).textContent = title;
+    caption.querySelector(`.${CARD_TEXT_CLASS_NAME}`).textContent = text;
+    caption.querySelector(`.${CARD_LINK_CLASS_NAME}`).setAttribute('href', link);
+    card.animate([
+        {opacity: 0},
+        {opacity: 1}
+    ], {
+        duration: 1000,
+        fill: 'forwards',
+        easing: 'ease-in'
+    })
+}
+
+// Execute on click of menu item
+function toggleCatDescriptionCard(event) {
+    // Prevent link's behaviour
+    event.preventDefault();
+    // Delete active class
+    menu.querySelector('.catalog-link__active').classList.remove('catalog-link__active');
+    // Get current item of the menu
+    const menuItem = event.target.closest('a.catalog-link');
+    // Add active class
+    menuItem.classList.add('catalog-link__active');
+    // Get catalog item for current menu item
+    let catalogItem = catalogItemsList.filter(item => item.id === +menuItem.dataset.id)[0];
+    // Update content
+    createCardPattern(catalogItem, card);
+}
+
+
+function createCards() {
+    const catalogItem = catalogItemsList[0];
```

Comment:
It's also not a blocker. But I think it would be better to rename this variable in order to associate it with the first element in the `catalogItemsList`